.github/workflows/ci-autowrap.yaml

```diff
@@ -42,6 +42,10 @@ jobs:
       - name: Upgrade pip version
         run: |
           python -m pip install -U pip
+      - name: Lint the code
+        run: |
+          python -m pip install black
+          python -m black . --check --diff
```

Comment:
Note: the `--check` flag doesn't apply any changes, it simply says if black would change any of the files. This effectively means the build will fail if the files don't conform to black standards.

The `--diff` flag will output the files that would change and how if one were to run `python -m black .` so the developer will know how to fix their failed PR.